common/src/main/scala/org/genivi/sota/marshalling/CirceInstances.scala

```diff
@@ -58,29 +55,22 @@ trait CirceInstances {
   implicit val uuidEncoder : Encoder[UUID] = Encoder[String].contramap(_.toString)
   implicit val uuidDecoder : Decoder[UUID] = Decoder[String].map(UUID.fromString)
 
-  implicit val dateTimeEncoder : Encoder[DateTime] =
-    Encoder.instance[DateTime]( x =>  Json.fromString( ISODateTimeFormat.dateTime().print(x)) )
+  implicit val dateTimeEncoder : Encoder[Instant] =
+    Encoder.instance[Instant]( x =>  Json.fromString( x.toString) )
 
-  implicit val dateTimeDecoder : Decoder[DateTime] = Decoder.instance { c =>
+  implicit val dateTimeDecoder : Decoder[Instant] = Decoder.instance { c =>
     c.focus.asString match {
       case None       => Xor.left(DecodingFailure("DataTime", c.history))
       case Some(date) =>
-        val parsers = List(ISODateTimeFormat.dateTimeNoMillis(), ISODateTimeFormat.dateTime())
-        tryParsers(date, parsers, DecodingFailure("DateTime", c.history))
+        tryParser(date, DecodingFailure("DateTime", c.history))
     }
   }
 
-  @tailrec
-  private def tryParsers(string: String, parsers: List[DateTimeFormatter],
-                         error: DecodingFailure): Xor[DecodingFailure, DateTime] = {
-    parsers match {
-      case parser :: otherParsers =>
-        try { Xor.right(parser.parseDateTime(string)) }
-        catch {
-          case t: IllegalArgumentException =>
-            tryParsers(string, otherParsers, error)
-        }
-      case Nil => Xor.left(error)
+  private def tryParser(string: String, error: DecodingFailure): Xor[DecodingFailure, Instant] = {
+    try { Xor.right(Instant.parse(string)) }
+    catch {
+      case t: IllegalArgumentException =>
+        Xor.left(error)
     }
   }
```

Comment:
Ah, ok. That's over HTTP. I tried (wrongly) to standardize on the (only) format that `java.time.Instant` knows about. I'll re-add the parsers and push again.